### PR TITLE
Add theme support in color Util

### PR DESCRIPTION
**Pull request by varun13169**
Add theme support in color Util

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -87,7 +87,7 @@ body {
 }
 
 .header__btn:hover {
-    background: var(--light-theme-color-hover-10pct);
+    background: var(--dui-light-theme-bg-opc-10pct);
 }
 
 .header__btn--download {}
```
Comment (sanu29) [inline styles.css:93]: No styles added thus it could be removed

**Review by sanu29 — APPROVED** [review]
Looks good for me to merge.

**Review by DivyanshSareen — APPROVED** [review]
good to merge

File: styles.css (hunk 1 above)
Comment (varun13169) [reply styles.css:93]: This is not part of the current PR.
Also, this is a placeholder for future use.

**Pull request merged**